deploy/kubernetes/overlays/stable/ecr/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+images:
+- name: amazon/aws-ebs-csi-driver
+  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver
+  newTag: v0.6.0
+- name: quay.io/k8scsi/csi-provisioner
+  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/csi-provisioner
+  newTag: v1.5.0
+- name: quay.io/k8scsi/csi-attacher
+  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/csi-attacher
+  newTag: v1.2.0
+- name: quay.io/k8scsi/livenessprobe
+  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/csi-liveness-probe
```

Comment:
for v2.0.0, I changed the name to just eks/livenessprobe to match the quay name. not applicable here, just noting for future when it gets bumped to v2.0.0.....